determined-ai/determined
# fix: bulk action bug in the old experiment table that cannot trigger bulk actions across pages

## Ticket

[ET-185]


## Description

Fix the bug that users cannot trigger batch actions when selected items are not in the page.
The new experiment table is already handled before this PR, so only handle it in the old experiment table


## Test Plan

- Turn on the old experiment table
- Check if bulk action are working across pages
  - e.g, 3 experiments are selected in page 1 and move to page 2 without selecting any experiments, and user still should be able to trigger bulk action in page 2
- No performance regression
   - no API should not be called at lot at once   


## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[ET-185]: https://hpe-aiatscale.atlassian.net/browse/ET-185?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
